pkg/utils/common.go

```diff
@@ -42,6 +42,7 @@ import (
 )
 
 const (
+	ClusterManagerName     = "KubeFleet"
```

Comment:
can we create a new package called "clustermanager" under util folder? really want to get rid of this util package as the "util" name does not provide any meaning.